webapp/src/pages/EditAvatarPage/EditAvatarPage.tsx

```diff
@@ -26,7 +29,11 @@ type ImgData = {
 };
 
 export default function EditAvatarPage() {
-  const { user } = useAuth();
+  const getCurrentUser = useCallback(
+    () => usersApi.userControllerGetCurrentUser(),
+    [],
+  );
```

Comment:
praise: :+1:  Thank you!

suggestion (if-minor): Since it's better to fetch the current user on the pages that needs this data, what do we think about converting the user from `useAuth` to a boolean, something like `isAuthed`/`isLoggedIn`?

Reply:
Seems like a good idea to me.

Reply:
done!